sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -128,6 +128,175 @@ case class MapKeys(child: Expression)
   override def prettyName: String = "map_keys"
 }
 
+@ExpressionDescription(
+  usage = """
+    _FUNC_(a1, a2, ...) - Returns a merged array containing in the N-th position the
+    N-th value of each array given.
```

Comment:
This description looks a bit confusing to me. How about `Returns a merged array of structs in which the N-th struct contains all N-th values of input arrays.`?